src/AudioUnit/AudioComponent.cs

```diff
@@ -224,6 +224,29 @@ public XamCore.AppKit.NSImage GetIcon ()
 			return new XamCore.AppKit.NSImage (AudioComponentGetIcon (handle));
 		}
 #endif
+		[NoWatch, NoTV, Mac (10,13), iOS (11,0)]
+		[DllImport (Constants.AudioUnitLibrary, EntryPoint = "AudioUnitExtensionSetComponentList")]
+		static extern int /* OSStatus */ SetComponentList (IntPtr /* CFString */ extensionIdentifier, /* CFArrayRef */ IntPtr audioComponentInfo);
+
+		[NoWatch, NoTV, Mac (10,13), iOS (11,0)]
+		[DllImport (Constants.AudioUnitLibrary, EntryPoint = "AudioUnitExtensionCopyComponentList")]
+		static extern /* CFArrayRef */ IntPtr CopyComponentList (IntPtr /* CFString */ extensionIdentifier);
+
+		[NoWatch, NoTV, Mac (10,13), iOS (11,0)]
+		public NSArray ComponentList {
+			get {
+				using (var cfString = new CFString (Name)) {
+					return new NSArray (CopyComponentList (cfString.Handle));
```

Comment:
I think you want `Runtime.GetNSObject<NSArray>(CopyComponentList (cfString.Handle));` instead or if we know the type inside the array we can use any of the NSArray.Fromxx to get a strongly typed array